n3fit/src/n3fit/checks.py

```diff
@@ -408,3 +408,11 @@ def check_fiatlux_pdfs_id(replicas, fiatlux, replica_path):
             raise CheckError(
                 f"Cannot generate a photon replica with id larger than the number of replicas of the PDFs set {luxset.name}:\nreplica id={max_id}, replicas of {luxset.name} = {pdfs_ids}"
             )
+
+@make_argcheck
+def check_multireplica_qed(replicas, fiatlux):
+    if fiatlux is not None:
+        if len(replicas) > 1:
+            raise CheckError(
+                "At the moment, running a multireplica QED fits is deprecated."
```

Comment:
It's not deprecated, it's not possible at all

Reply:
Well, actually it is possible. The problem is that all the photons will be computed in series so it will take around 30 min * 100 replicas = 50 hours

Reply:
```suggestion
                "At the moment, running a multireplica QED fits is not allowed."
```

This is correct in any case.